flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -176,5 +176,23 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table_2.10</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-scala_2.10</artifactId>
```

Comment:
I still don't understand this :/

If neither flink-table nor we actually depend on flink-scala, nor flink-scala-streaming then we don't need the dependency. If the user wants to work against the scala api he has to specify this dependency himself.